tests/flow/test_temporal.py

```
@@ -0,0 +1,636 @@
+          #('2015W30',    '2015-07-20'),
+          ('2015-07',    '2015-07-01'),
+          #('2015-202',   '2015-07-21'),
+          #('2015W302',   '2015-07-21'),
+          ('2015-W30',   '2015-07-20'),
+          ('20150721',   '2015-07-21'),
+          ('2015-07-21', '2015-07-21'),
+          ('2015-W30-2', '2015-07-21')
+        ]
+
+        for str_input, expected in test_cases:
+            query = f"RETURN date($str)"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)